docs/development/extensions-core/kafka-ingestion.md

```diff
@@ -40,6 +40,8 @@ This service is provided in the `druid-kafka-indexing-service` core Apache Druid
 > In addition, users could set `isolation.level` `read_uncommitted` in `consumerProperties`, if don't need Druid to consume transactional topics or need Druid to consume older versions of Kafka.
 > Make sure offsets are sequential, since there is no offset gap check in Druid anymore.
 
+> If your Kafka cluster enables ACLs, you can set `group.id` in `consumerProperties` to override the defualt auto generated group id.
```

Comment:
After updating the doc, you may need to add the `ACLs` to the `.spelling` file, and then please use `default` instead of `defualt`.